soscleaner/soscleaner.py

```diff
@@ -417,7 +417,7 @@ def _prime_userdb(self):
         """
 
         try:
-            new_user = "obfuscateduser%s" % self.user_count
+            new_user = "obfuscateduser%s" % randint(1,1000000)
```

Comment:
We don't seems to save much but was curious to see the difference if we decrease the randint range.

```
$ python3 -m timeit -s 'import random' 'random.randint(1,1000000)'
1000000 loops, best of 3: 0.814 usec per loop

$ python3 -m timeit -s 'import random' 'random.randint(1,1000)'
1000000 loops, best of 3: 0.788 usec per loop

$ python3 -m timeit -s 'import random' 'random.randint(1,100)'
1000000 loops, best of 3: 0.772 usec per loop
```

Reply:
65536 POSIX users, and lots of apps use random higher apps (especially for container platform hosts).
I think I'm ok with .026 seconds to make sure we don't hit some corner case down the road.

Pulled this PR because I got tests working and have a second commit.